src/NeoDemo/NeoDemo.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
I don't see why we would jump to net5 when net6 is the latest LTS.

Reply:
It was not part of this PR, but I can submit .NET 6.0 migration PR as well :D